From 2385dbae3f5f20c167129cbd836e6d5f9fc21783 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 12 Aug 2021 19:39:51 +0200
Subject: [PATCH 1/2] redirect tomlplusplus-include to global library path

---
 include/pytomlpp/pytomlpp.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/pytomlpp/pytomlpp.hpp b/include/pytomlpp/pytomlpp.hpp
index b5020f5..194a3b2 100644
--- a/include/pytomlpp/pytomlpp.hpp
+++ b/include/pytomlpp/pytomlpp.hpp
@@ -32,7 +32,7 @@
 #endif
 #include "compilers.hpp"
 PYTOMLPP_PUSH_OPTIMIZATIONS;
-#include <tomlplusplus/include/toml++/toml.h>
+#include <toml++/toml.h>
 PYTOMLPP_POP_OPTIMIZATIONS;
 
 // namespace and type forward declarations
-- 
2.38.1.windows.1

