From 650bf56fa1748b1a1a442aa322027547c2b7186c Mon Sep 17 00:00:00 2001
From: Bob Fang <dorafmon@gmail.com>
Date: Sat, 29 Nov 2025 16:48:24 -0500
Subject: [PATCH] unpin pybind11 for pypy

---
 setup.cfg | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index cb1e4ba..375ed1d 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -29,8 +29,7 @@ classifiers =
 keywords = toml, parser, serilization, deserialization, serdes
 
 [options]
-setup_requires =
-    pybind11~=2.10
+setup_requires = pybind11
 zip_safe = false
 packages = find:
 package_dir = =src
-- 
2.50.1 (Apple Git-155)

